.vscode/launch.json

```diff
@@ -1,14 +1,46 @@
 {
-    // Use IntelliSense to learn about possible attributes.
-    // Hover to view descriptions of existing attributes.
-    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-    "version": "0.2.0",
-    "configurations": [
-        {
-            "name": "Attach to Process (from list)",
-            "type": "go",
-            "request": "attach",
-            "mode": "local"
-          },
-    ]
-}
\ No newline at end of file
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Index folder",
+      "type": "go",
+      "request": "launch",
+      "mode": "auto",
+      "program": "cmd/zoekt-git-index",
+      "cwd": "${workspaceFolder}",
+      "args": ["${input:path}"]
```

Comment:
need to specify input:indexPath here as well

Reply:
I dont think this command lets you pass index path